Mathlib/Order/Defs/PartialOrder.lean

```diff
@@ -104,6 +104,9 @@ def decidableLTOfDecidableLE [DecidableRel (α := α) (· ≤ ·)] : DecidableRe
       else isTrue <| lt_of_le_not_le hab hba
     else isFalse fun hab' => hab (le_of_lt hab')
 
+lemma not_lt_iff_not_le_or_ge : ¬a < b ↔ ¬a ≤ b ∨ b ≤ a := by
+  rw [lt_iff_le_not_le, Classical.not_and_iff_or_not_not, Classical.not_not]
```

Comment:
I think this should work:
```suggestion
  rw [lt_iff_le_not_le, not_and_or, Classical.not_not]
```
Otherwise LGTM.

Reply:
This imports `Mathlib.Logic.Basic`. It seems to only import the 1000 lines in the file, and the rest are just tactics - I'm unsure if that's a 'good' large import in mathlib4.

Reply:
In any case your original proof avoids this large import issue, so I retract my suggestion.